restful.module

```diff
@@ -786,3 +786,18 @@ function restful_is_user_switched() {
   return TRUE;
 }
 
+/**
+ * Get the query string from php Input.
+ *
+ * @return string
+ *   The query string from request.
+ */
+function _restful_get_query_string_from_php_input() {
+    $query_string = &drupal_static(__FUNCTION__);
+    if (!isset($query_string)) {
+      // In php 5.6 and below php://input can only be read once.
```

Comment:
I just read about this :open_mouth: 

![- manual 2015-04-14 14-30-38](https://cloud.githubusercontent.com/assets/1140906/7136617/e859ad26-e2b2-11e4-94ff-4f6dce2eb759.png)